packit/api.py

```diff
@@ -778,6 +780,44 @@ def check_version_distance(
                     "to skip this check.",
                 )
 
+    @staticmethod
+    def get_upstream_release_monitoring_bug(
+        package_name: str,
+        version: str,
+    ) -> Optional[str]:
+        """
+        Obtain the bug created by Upstream Release Monitoring
+        about the new upstream release matching the package_name
+        and the version via Bugzilla API.
+
+        Returns bugzilla if found in format 'rhbz#{id}'
+        """
+        bzapi = bugzilla.Bugzilla(BUGZILLA_HOSTNAME)
```

Comment:
This can be unauthenticated? Should we expect some issues when running more queries?

Reply:
from https://bugzilla.readthedocs.io/en/latest/api/core/v1/general.html#authentication:

> Some methods do not require you to log in. An example of this is [Get Bug](https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#rest-single-bug). However, authenticating yourself allows you to see non-public information, for example, a bug that is not publicly visible.

searching works for me without authentication and I haven't found anything about rate-limiting, so hopefully this will be enough